test/JDBC/upgrade/master/schedule

```diff
@@ -107,6 +107,7 @@ BABEL-CHECK-CONSTRAINT
 babel_datetime
 BABEL-EXECUTE_AS_CALLER
 BABEL-EXTENDEDPROPERTY
+BABEL-EXTENDEDPROPERTY-v2
```

Comment:
This test is probably going to fail in database-level dump/restore github action since we only dump `master` database in that action and we don't dump logins as well so it's better to not add this test here directly. Although this test should already be getting executed in instance-level dump/restore (in this we dump everything).
Is it possible to write a minimal test in which we just create objects in master database so that we can add that test here?

Reply:
Now `BABEL-EXTENDEDPROPERTY-v2` is used to test permissions between sysadmin and normal user. sysadmin can add, update, drop, list extended property of six objects, while normal user can only list extended property if he has SELECT privilege.
So, is it necessary to add an individual minimal test only in `test/JDBC/upgrade/master/schedule`? If the answer is yes, what kind of test should I add in this minimal test?